packages/@react-spectrum/table/docs/TableView.mdx

```diff
@@ -560,6 +560,118 @@ function AsyncSortTable() {
+width to a minimum and maximum value respectively. TableView prioritizes columns with defined widths and divides the remaining space evenly amongst the other columns.
+
+The example below illustrates how each of the aforementioned column width properties affects their respective column's resize behavior.
+
+```tsx example
+<TableView
+  aria-label="TableView with resizable columns"
+  maxWidth={320}
+  height={200} >
+  <TableHeader>
+    {/*- begin highlight -*/}
+    <Column key="file" allowsResizing maxWidth={500}>File Name</Column>
+    <Column key="size" width={80}>Size</Column>
+    <Column key="date" allowsResizing minWidth={100}>Date Modified</Column>
+    {/*- end highlight -*/}
+  </TableHeader>
+  <TableBody>
+    <Row>
+      <Cell>2022-Roadmap-Proposal-Revision-012822-Copy(2)</Cell>
+      <Cell>214 KB</Cell>
+      <Cell>November 27, 2022 at 4:56PM</Cell>
+    </Row>
+    <Row>
+      <Cell>62259692_p0_master1200</Cell>
+      <Cell>120 KB</Cell>
+      <Cell>January 27, 2021 at 1:56AM</Cell>
+    </Row>
+  </TableBody>
+</TableView>
+```
+
+### Resize events
+
+TableView accepts an `onResize` prop which is triggered whenever a column resizer is moved by the user. This can be used in combination with the `width` prop to update a Column's width in a controlled fashion. TableView also accepts
+an `onResizeEnd` props, which is triggered when the user finishes a column resize operation. Both events receive a Map object containing the widths of every column in the TableView.
+
+The example below uses `onResize` to update each of the TableView's controlled column widths. It also saves the finalized column widths to `localStorage` in `onResizeEnd`, allowing the TableView's state to be preserved between
+page loads and refreshes.
+
+```tsx example
+let items = [
+  {id: '1', file: '2022-Roadmap-Proposal-Revision-012822-Copy(2)', size: '214 KB', date: 'November 27, 2022 at 4:56PM'},
+  {id: '2', file: '62259692_p0_master1200', size: '120 KB', date: 'January 27, 2021 at 1:56AM'}
+];
+
+let columnsData = [
+  {name: 'File Name', id: 'file', width: '1fr'},
+  {name: 'Size', id: 'size', width: 80},
+  {name: 'Date', id: 'date', width: 100}
+];
+
+function ResizableTable() {
+  /*- begin highlight -*/
+  let [columns, setColumns] = React.useState(() => {
+    let localStorageWidths = localStorage.getItem('RSPWidths');
+    if (localStorageWidths) {
+      let widths = JSON.parse(localStorageWidths);
+      return columnsData.map(col => ({...col, width: widths[col.id]}));
+    } else {
+      return columnsData;
+    }
+  });
+
+  let onResize = (widths) => {
+    setColumns(columns => columns.map(col => ({...col, width: widths.get(col.id)})));
+  };
+
+  let onResizeEnd = (widths) => {
+    localStorage.setItem('RSPWidths', JSON.stringify(Object.fromEntries(widths)));
+  };
+  /*- end highlight -*/
+  return (
+    <Flex direction="column">
```

Comment:
What's the extra flex for?

Reply:
Ah derp, hold over from when the example had `pre` elements to show what widths were being provided to the event handlers, thanks for catching